### PR TITLE
Create gaia.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/85814/style-request-gaia/p1

### CODE REVIEW

File: gaia.csl [hunk 1]
```diff
@@ -0,0 +1,527 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+  <!-- Polyglot; journal publishes in German and English -->
+  <info>
+    <title>GAIA</title>
+    <id>http://www.zotero.org/styles/gaia</id>
+    <link href="http://www.zotero.org/styles/gaia" rel="self"/>
+    <link href="http://www.zotero.org/styles/business-ethics-a-european-review" rel="template"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1111/(ISSN)1467-8608/homepage/ForAuthors.html" rel="documentation"/>
```
Comment (rmzelle) [inline gaia.csl:9]: @POBrien333, documentation link is still that from the template.

File: gaia.csl [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="humanities"/>
+    <issn>0940-5550</issn>
+    <eissn>2625-5413</eissn>
```
Comment (rmzelle) [inline gaia.csl:16]: ISSNs too.

Comment (POBrien333) [reply gaia.csl:16]: No, ISSNs are correct.

**Comment by rmzelle** [comment]
Thanks!

**Comment by rmzelle** [comment]
:christmas_tree:

**Pull request merged**